.github/workflows/truework-sdk-ruby-ci.yml

```diff
@@ -14,7 +14,7 @@ jobs:
     steps:
     - uses: actions/checkout@v2
     - name: Set up Ruby
-      uses: ruby/setup-ruby@ec106b438a1ff6ff109590de34ddc62c540232e0
+      uses: ruby/setup-ruby@v1
```

Comment:
This is the recommended way to use this action: https://github.com/ruby/setup-ruby#versioning